src/pages/signin/SignInPage.js

```diff
@@ -273,4 +276,5 @@ export default withOnyx({
     We use that function to prevent repeating code that checks which client is the leader.
     */
     activeClients: {key: ONYXKEYS.ACTIVE_CLIENTS},
+    betas: {key: ONYXKEYS.BETAS},
```

Comment:
Can't we use `withBetas` instead?

Reply:
ooh yes you're rihgt

Reply:
I only see `src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js` using it that was atm